Source/Core/VideoCommon/FrameDump.cpp

```diff
@@ -336,10 +336,17 @@ void FrameDump::ProcessPackets()
 {
   while (true)
   {
-    AVPacket pkt;
-    av_init_packet(&pkt);
+    std::unique_ptr<AVPacket, std::function<void(AVPacket*)>> pkt =
```

Comment:
Maybe use auto here like:

```
auto pkt = ...
```

to reduce the duplication of the type a little.

As an aside, perhaps this should be moved out of the loop so the packet is only allocated once and then reused throughout the iterations to avoid heap churn a little bit.